### PR TITLE
all: tick version to 1.2.0

### PR DESCRIPTION
**Pull request by jeffomatic**

### CODE REVIEW

**Comment by jeffomatic** [comment]
This is WIP.

**Comment by jeffomatic** [comment]
PTAL. Barring mishaps, the resulting commit for this PR should be `1.2.0`-taggable for all artifacts except for `chain-core-mac-1.2.0`, which requires an additional build step.

**Comment by kr** [comment]
:eyes:

File: sdk/ruby/CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,52 @@ This changelog is for the 1.2 branch of the Ruby SDK. Older versions:
...
+
+### New features
+
+#### Authentication and authorization
+
+The `Chain::AuthorizationGrant` class allows you to programmatically register credentials that have well-scoped access to different parts of the Chain Core API. See the [documentation](https://chain.com/docs/1.2/core/learn-more/authentication-and-authorization) for code samples, as well as a detailed introduction to authentication and authorization in Chain Core.
+
+#### TLS configuration
+
+The SDK now features flexible TLS configuration, allowing you to accept server certificates outside of the main X.509 PKI, and to authenticate requests to Chain Core using client X.509 certificates.
```
Comment (kr) [inline sdk/ruby/CHANGELOG.md]: nit: s/main/public/

File: sdk/java/CHANGELOG.md [hunk 2]
```diff
@@ -5,11 +5,49 @@ This changelog is for the 1.2 branch of the Java SDK. Older versions:
...
+
+#### Authentication and authorization
+
+The `AccessToken` and `AuthorizationGrant` classes have been added to enable programmatic control over authentication and authorization of requests made to Chain Core. This lets you generate credentials that have well-scoped access to different parts of the Chain Core API.
+
+See the [documentation](https://chain.com/docs/1.2/core/learn-more/authentication-and-authorization) for code samples, as well as a detailed introduction to authentication and authorization in Chain Core.
+
+#### TLS configuration
+
+The SDK now features flexible TLS configuration, allowing you to accept server certificates outside of the main X.509 PKI, and to authenticate requests to Chain Core using client X.509 certificates.
```
Comment (kr) [inline sdk/java/CHANGELOG.md]: nit: s/main/public/

File: docs/core/reference/changelog.md [hunk 3]
```diff
@@ -10,8 +10,8 @@ For updates to subpackages, see below:
 - [Node.js SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/node/CHANGELOG.md)
 - [Ruby SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/ruby/CHANGELOG.md)
 
-<a name="1.2rc2"></a>
-## 1.2rc2 (May 4, 2017)
+<a name="1.2.0"></a>
+## 1.2.0 (May 4, 2017)
```
Comment (kr) [inline docs/core/reference/changelog.md]: May 4 or 12?

File: sdk/node/CHANGELOG.md [hunk 4]
```diff
@@ -5,9 +5,39 @@ This changelog is for the 1.2 branch of the Node.js SDK. Older versions:
...
+### Breaking changes
+
+#### Control programs removed in favor of receivers
+
+To perform cross-core transfers, you should use receivers instead of control program. The pattern for using receivers is nearly identical to using control programs. The key differences are:
+
+- Receivers are created via the following pattern:
+
+    ```
+    client.accounts.create_receiver(account_id: '...')
```
Comment (chrisgarvin) [inline sdk/node/CHANGELOG.md]: Think this needs to be js'ified: `createReceiver`

File: sdk/node/CHANGELOG.md [hunk 5]
```diff
@@ -5,9 +5,39 @@ This changelog is for the 1.2 branch of the Node.js SDK. Older versions:
...
+    ```
+
+    This differs slightly to the removed `client.control_programs.create()` pattern.
```
Comment (chrisgarvin) [inline sdk/node/CHANGELOG.md]: `client.controlPrograms.create()`

**Pull request merged**